playwright/search-page/search.spec.ts

```diff
@@ -0,0 +1,175 @@
+import { test } from "@playwright/test";
+import { SearchPage } from "./search-page";
+
+test.describe("EDD Request Process", () => {
+  test("Begin the EDD request process  and the delivery locations page is displayed", async ({
+    page,
+  }) => {
+    const searchPage = new SearchPage(page);
+    await searchPage.navigateToSearchPage();
+    await searchPage.fillSearchBox("Africa");
+    await searchPage.clickSearchButton();
+    await searchPage.clickRequestableCheckbox();
+    await searchPage.verifyFirstLoginForOptionsButtonVisible();
+  });
+});
+
+test.describe("Government Documents Filter", () => {
+  test("Search for government documents and the first search result displays the United States as the author", async ({
+    page,
+  }) => {
+    const govDocsPage = new SearchPage(page);
+    await govDocsPage.navigateToSearchPage();
+    await govDocsPage.fillSearchBox("swimming");
+    await govDocsPage.clickSearchButton();
+    await govDocsPage.clickGovernmentDocumentsCheckbox();
+    await govDocsPage.verifyFirstGovernmentDocumentAuthorVisible();
+  });
+});
+
+test.describe("Keyword Search", () => {
+  test("Search for a keyword and the first search result contains the keyword", async ({
+    page,
+  }) => {
+    const searchPage = new SearchPage(page);
+    await searchPage.navigateToSearchPage();
+    await searchPage.fillSearchBox("IBM 1401");
+    await searchPage.clickSearchButton();
+    await searchPage.verifyFirstSearchResultKeywordVisible();
+  });
+});
+
+test.describe("Language Filter", () => {
```

Comment:
@kylevillegas93 these tests are timing out and some in the item-detail.spec

You can see them here: https://github.com/NYPL/sfr-bookfinder-front-end/actions/runs/12604412935/job/35131274366?pr=573